src/Middleware/CORS/test/testassets/TestDestination/StartupWithoutEndpointRouting.cs

```diff
@@ -12,7 +13,7 @@ namespace SampleDestination
 {
     public class StartupWithoutEndpointRouting
     {
-        private static readonly string DefaultAllowedOrigin = $"http://{Dns.GetHostName()}:9001";
+        private static readonly string DefaultAllowedOrigin = $"http://localhost:{Environment.GetEnvironmentVariable("ORIGIN_PORT")}";
```

Comment:
Missed this bit in my initial review. Could we use `Dns.GetHostName()`? Chrome will sometimes not do Cors if it's running on localhost. Using the hostname avoids this.